dangerzone/conversion/common.py

```diff
@@ -19,8 +19,27 @@
 
 def running_on_qubes() -> bool:
     # https://www.qubes-os.org/faq/#what-is-the-canonical-way-to-detect-qubes-vm
-    if os.environ.get("DZ_USE_CONTAINERS", "0") == "0":
-        return os.path.exists("/usr/share/qubes/marker-vm")
+    return os.path.exists("/usr/share/qubes/marker-vm")
+
+
+def qubes_native_conversion() -> bool:
+    """Returns True if the conversion should be run using Qubes OS's diposable
+    VMs and False if not."""
+    if running_on_qubes():
+        if os.environ.get("VM_CONVERSION", "0") == "1":
+            return True
+        elif not getattr(sys, "dangerzone_dev", False):
```

Comment:
I'm not sure I get why we need to check if the code runs in dev mode. Can you explain it please?

Reply:
This is the case where the user installed the package. We want to know if they installed the package with a qubes built target (i.e. if it was using `build-rpm.py --qubes`) or if it is just a fedora version, which just happens to be running on Qubes.

But now looking again at the logic it feels a bit convoluted.

Reply:
Right, but that's not related with `dangerzone_dev`, if I understand correctly. I mean, replacing `elif not getattr(sys, "dangerzone_dev", False):` with `else:` would be effectively the same. I believe the only way this `elif` affects us is:

* if a developer runs Dangerzone in Qubes,
* **and** `QUBES_CONVERSION` is not defined or false,
* **then** it will always use the container isolation provider, without checking if `containers.tar.gz` exists.

Could it be that the above logic is necessary in your environment? If not, I suggest removing it, because it's indeed convoluted.

Reply:
(facepalm) you're right. I've simplified this logic now. Please check if it's correct now.